ios/FluentUI/Pill Button Bar/PillButton.swift

```diff
@@ -108,12 +113,12 @@ open class PillButton: UIButton {
     private func setupView() {
         if #available(iOS 15.0, *) {
             var configuration = UIButton.Configuration.plain()
-            configuration.attributedTitle = AttributedString(pillBarItem.title)
             configuration.contentInsets = NSDirectionalEdgeInsets(top: Constants.topInset,
                                                                   leading: Constants.horizontalInset,
                                                                   bottom: Constants.bottomInset,
                                                                   trailing: Constants.horizontalInset)
             self.configuration = configuration
+            updateAttributedTitle()
```

Comment:
From the change notes, is the order in which you call this important? Relative to the line before? If so, good to add a comment explaining for which calls the order matters and what bug one should look for if they try to change the order in the future